components/input/docs/partials/api.md

```diff
@@ -212,6 +193,25 @@ The `setCustomValidityRangeUnderflow` attribute may optionally be used in combin
 
 </auro-accordion>
 
+### Max Length <a name="maxLength"></a>
```

Comment:
Why did this move? Wasn't it in alphabetical order before?

Reply:
It was not in alphabetical order. I kept getting confused when looking at the `maxLength` example right next to the `max` example so I shuffled it to avoid confusion and make the flow of demos more readable.